portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,16 +32,23 @@
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  int maxNumComments = 1;
```

Comment:
This is an interesting artifact of servlets existing for the lifetime of the container - changes to this member are going to be visible to all users. If user A requests 5 max comments and user b requests 10 max comments, the latter one will always win if the requests are interleaved.

Can you rethink this s.t. the doGet method accepts a max number of comments in the query string?

Reply:
+1. the max comment count could be part of the get request.

Reply:
sounds good! I'll give this some thought :)

Reply:
fixed I think!